verl/utils/tracking.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, project_name, experiment_name, default_backend: Union[str, Li
 
         if 'tracking' in default_backend or 'wandb' in default_backend:
             import wandb
-            wandb.init(project=project_name, name=experiment_name, config=config, resume='allow')
+            wandb.init(project=project_name, name=experiment_name, id=experiment_name, config=config, resume='allow')
```

Comment:
Weve discussed this in the older PR. If you want this, please put the fix you proposed in the earlier pr.